pkg/cmd/server/start.go

```diff
@@ -313,6 +314,11 @@ func (o *ServerOptions) RunServer(stopCh <-chan struct{}, title, version string,
 	if err != nil {
 		return err
 	}
+
+	aggregatedAPIServerConfig.RecommendedConfig.Config.OpenAPIConfig = genericapiserver.DefaultOpenAPIConfig(GetOpenApiDefinition, openapinamer.NewDefinitionNamer(builders.Scheme))
+	aggregatedAPIServerConfig.RecommendedConfig.Config.OpenAPIConfig.Info.Title = title
+	aggregatedAPIServerConfig.RecommendedConfig.Config.OpenAPIConfig.Info.Version = version
+
 	genericConfig := aggregatedAPIServerConfig.RecommendedConfig.Config
```

Comment:
```suggestion
      genericConfig := &aggregatedAPIServerConfig.RecommendedConfig.Config
```

thank you for fixing this. how you think about another pass? @jeremyxu2010

Reply:
done as you suggested @yue9944882